Sources/SwiftDocC/Model/Identifier.swift

```diff
@@ -639,7 +639,7 @@ private extension CharacterSet {
 ///
 /// If this step is not performed, the disallowed characters are instead percent escape encoded, which is less readable.
 /// For example, a fragment like `"#hello world"` is converted to `"#hello-world"` instead of `"#hello%20world"`.
-func urlReadableFragment(_ fragment: String) -> String {
+func urlReadableFragment<S: StringProtocol>(_ fragment: S) -> String {
```

Comment:
I don't remember what version of swift we aim to support, would `some StringProtocol` be acceptable here

Reply:
Not yet, we support Swift 5.5 and the `some` keyword was added in Swift 5.7